vue-gtag.d.ts

```diff
@@ -0,0 +1,78 @@
+declare module 'vue-gtag' {
+  import _Vue, { PluginFunction } from 'vue';
+  import VueRouter from 'vue-router';
+
+  export interface PageView {
+    page_path: string;
```

Comment:
missing `page_location` and `page_title` as options for this call

Reply:
Added these missing props